src/Client/SceneManager.cpp

```diff
@@ -31,22 +26,6 @@ static void initRaylib()
     InitWindow(screenWidth, screenHeight, "R-Bus");
     SetTargetFPS(fps);
     InitAudioDevice();
-    Registry::getInstance().addEntity();
```

Comment:
this deleted code is some poc, moved in a system for more poc things like proving that system can auto delete after creating player entity.